usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_geography.md

```diff
@@ -0,0 +1,305 @@
+## [Spending by Geography](#spending-by-geography)
+**Route:** `/api/v2/search/spending_by_geography/`
+
+**Method:** `POST`
+
+This route takes award filters, and returns spending by state code, county code, or congressional district code.
+
+### Request
+scope: Scope defines what location is used for filtering the aggregated_amount.
```

Comment:
scope: What type of data will be returned. Must be either: `place_of_performance` or `recipient_location`

I assume there aren't other possible values.

Reply:
those are the only 2
```
 model_dict = {
            'place_of_performance': 'pop',
            'recipient_location': 'recipient_location'
        }```